app/src/main/java/com/example/playlistmaker/player/ui/PlayerFragment.kt

```diff
@@ -37,13 +39,17 @@ class PlayerFragment : Fragment() {
         super.onViewCreated(view, savedInstanceState)
 
         binding.playbackButton.setOnClickListener {
-            viewModel.playbackControl()
+            viewModel.togglePlayback()
         }
 
         binding.navigation.setNavigationOnClickListener {
             findNavController().popBackStack()
         }
 
+        binding.addToFavouritesFab.setOnClickListener {
+            viewModel.saveOrDeleteFromFavourites()
```

Comment:
🍏 Рекомендую скрыть от фрагмента логику вьюмодели через другое наименование. Например, так:
```suggestion
            viewModel.onFavouritesButtonClick()
```
Возможно, в будущем вьюмодель будет делать что-то совсем другое при нажатии на эту кнопку, но на фрагмент это не повлияет)